docs/guides/auth.md

```diff
@@ -302,7 +302,26 @@ $client = new helloworld\GreeterClient(
   new Grpc\BaseStub('localhost:50051', []));
 ...
 
-// Authenticating with Google
+// Authenticate using JWT access token (recommended approach)
+function updateAuthMetadataCallback($context)
+{
+    $authUri = $context->service_url;
+    $methodName = $context->method_name;
+    $auth_credentials = ApplicationDefaultCredentials::getCredentials();
+
+    return $auth_credentials->updateMetadata($metadata = [], $authUri);
+}
+$opts = [
+  'credentials' => Grpc\Credentials::createSsl(file_get_contents('ca.pem'));
+];
+
+$client = new helloworld\GreeterClient('localhost:50051', $opts);
```

Comment:
@jboeuf Thanks for the feedback. I lifted this code example from the interop tests so it may not be the best for documentation. How about if I change the example to this?

```
// Authenticate using scopeless credentials (recommended approach)
function updateAuthMetadataCallback($context)
{
    $authUri = $context->service_url;
    $methodName = $context->method_name;
    $auth_credentials = ApplicationDefaultCredentials::getCredentials();

    return $auth_credentials->updateMetadata($metadata = [], $authUri);
}
$credentials = Grpc\ChannelCredentials::createComposite(
    Grpc\Credentials::createSsl(file_get_contents('ca.pem')),
    Grpc\CallCredentials::createFromPlugin('updateAuthMetadataCallback')
);
$opts = [
  'credentials' =>  $credentials
];

$client = new helloworld\GreeterClient('greeter.googleapis.com', $opts);
```

I hope that's cleaner.